src/include/settings/settings_defs.h

```diff
@@ -279,4 +298,22 @@ SETTING_string(
     true,
     terrier::settings::Callbacks::NoOp
 )
+
+SETTING_int(
+    num_parallel_execution_threads,
+    "Number of threads for parallel query execution (default: 0)",
+    0,
```

Comment:
What does 0 mean... no thread to execute the parallel task? It's a bit confusing to me.

Reply:
I changed this to 1 (which matches the behavior of 0 on my system).